usrsctplib/netinet/sctp_callout.c

```diff
@@ -218,7 +218,7 @@ sctp_os_timer_stop(sctp_os_timer_t *c)
 }
 
 void
-sctp_handle_tick(int delta)
+sctp_handle_tick(int elapsed_ticks)
 {
```

Comment:
Not sure what you gain by this...

Reply:
@tuexen I think this name better explain the meaning, but that's of course my subjective opinion... And I'd liked to start with very tiny contribution before doing bigger changes :)